packages/fxa-auth-server/lib/server.js

```diff
@@ -427,6 +430,35 @@ async function create(log, error, config, routes, db, statsd) {
 
   server.auth.strategy('pubsub', 'jwt', pubsubAuth.strategy(config));
 
+  const swaggerOptions = {
+    info: {
+      title: 'Firefox Accounts Authentication Server API',
+      description:
+        '<p><b>WARNING</b>: This information may not be up-to-date, use it at your own risk. It may be worth verifying information in the source code before acting on anything you read here.</p>' +
+        '<p>This document provides protocol-level details of the Firefox Accounts auth server API. For a prose description of the client/server protocol and details on how each parameter is derived, see the <a href="https://wiki.mozilla.org/Identity/AttachedServices/KeyServerProtocol"><b>API design document</b></a>. For a reference client implementation, see <a href="https://github.com/mozilla/fxa/tree/main/packages/fxa-auth-client"><code>fxa-auth-client</code></a>.</p>' +
+        '<ul><li><a href="https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#overview">Overview</a></li>' +
+        '<li><a href="https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#url-structure">URL structure</a></li>' +
+        '<li><a href="https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#request-format">Request format</a></li>' +
+        '<li><a href="https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#response-format">Response format</a></li>' +
+        '<ul><li><a href="https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#defined-errors">Defined errors</a></li>' +
+        '<li><a href="https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#responses-from-intermediary-servers">Responses from intermediary servers</a></li></ul>' +
```

Comment:
This looks like a bit of a weird nested list, but isn't nested correctly.
Not sure if we need to remove the wrapping `<ul>..</ul>` or else nest it inside one of the other `<li>..</li>` tags.